scripts/ginkgo.coverage.sh

```diff
@@ -25,6 +25,8 @@ hash goveralls 2>/dev/null || go get github.com/mattn/goveralls
 hash godir 2>/dev/null || go get github.com/Masterminds/godir
 
 generate_cover_data() {
+  ginkgo version
+  git diff --exit-code --quiet go.mod go.sum vendor/modules.txt || go mod vendor
```

Comment:
This is the key change to unblock CI, we want to figure out why `make coverage` gets itself into a place where the above files are dirty.